code/config.py

```diff
@@ -244,22 +245,25 @@ def time_series_parameters(param):
     :rtype: dict
     """
     # Quantiles for time series
-    param["quantiles"] = [100, 90, 80, 70, 60, 50, 40, 30, 20, 10, 0]
+    param["quantiles"] = [90, 70, 60, 40]
```

Comment:
can you undo this please

Reply:
yes, I missed it when resetting the config.py